src/features/broker-protection/extractors/profile-url.js

```diff
@@ -0,0 +1,28 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { Extractor } from '../types.js'
+import { getIdFromProfileUrl } from '../actions/extract.js'
```

Comment:
Should we just move `getIdFromProfileUrl` into this file? I don't think it's called from anywhere else.

Reply:
yep - will follow up :)